src/presentation/react/core/components/create-package-from-file-dialog/CreatePackageFromFileDialog.tsx

```diff
@@ -57,6 +58,7 @@ export const CreatePackageFromFileDialog: React.FC<CreatePackageFromFileDialogPr
 
     useEffect(() => {
         compositionRoot.user.current().then(({ userGroups }) => setUserGroups(userGroups));
+        compositionRoot.dhisReleases.getSupportedDhisVersions().then(versions => setDhisSupportedVersions(versions));
```

Comment:
Not a big deal, but since we're using this method here and in `PackagesExtendCompatibilityDialog.tsx` maybe we can extract it into a custom hook:

```ts

export function useGetSupportedVersion() {
    React.useEffect(() => {
        compositionRoot.dhisReleases.getSupportedDhisVersions().then(setDhisSupportedVersions);
    }, [compositionRoot]);
}

```

this way if we need to fetch the versions in other components we can reuse the same hook. What do you think?